nncf/quantization/algorithms/accuracy_control/algorithm.py

```diff
@@ -134,18 +145,24 @@ def restore_accuracy(
                 validate the provided model.
             The function should return the value of the metric with the following meaning:
             A higher value corresponds to better performance of the model.
-        :return: The quantized model whose metric `final_metric` is satisfied the following condition
-
-            initial_metric - final_metric <= max_drop.
+        :return: The quantized model whose metric `final_metric` is satisfied
+            the maximum accuracy drop condition.
         """
+        if self.drop_type == DropType.ABSOLUTE:
+            max_absolute_drop = self.max_drop
```

Comment:
Maybe it's better to leave just the `max_drop` variable, to not confuse anyone.

Reply:
done